cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/DefaultSecureStoreServiceTest.java

```diff
@@ -69,23 +70,30 @@
 
   @BeforeClass
   public static void setup() throws Exception {
-    Injector injector = Guice.createInjector(new AppFabricTestModule(createCConf()));
-    secureStoreService = injector.getInstance(SecureStoreService.class);
+    CConfiguration cConf = createCConf();
+    Injector injector = AppFabricTestHelper.getInjector(cConf);
     authorizer = injector.getInstance(AuthorizerInstantiator.class).get();
+    authorizer.grant(new InstanceId(cConf.get(Constants.INSTANCE_NAME)), ALICE, Collections.singleton(Action.ADMIN));
+    SecurityRequestContext.setUserId(ALICE.getName());
+    injector.getInstance(NamespaceAdmin.class).create(NamespaceMeta.DEFAULT);
+    secureStoreService = injector.getInstance(SecureStoreService.class);
+    authorizer.revoke(NamespaceId.DEFAULT, ALICE, Collections.singleton(Action.ALL));
+    RemoteSystemOperationsService remoteSystemOperationsService =
```

Comment:
I get this now 
java.lang.IllegalStateException: Cannot use authorization enforcement service because it has not been started. Its current state is NEW.